LosAngelesDataProject/LADP
# Map markers

- (Map Render)[
![image](https://github.com/LosAngelesDataProject/LADP/assets/149129171/5dd239fc-da0a-474e-9d02-b9f2acdd86fb)
]

# Modifications
- Added dynamic marker to the map module 
- Added map to home component in map of the static map img

# To test
-  open the branch and run the application, Map component should render on Home page.
